src/form/en_acr.jl

```diff
@@ -0,0 +1,839 @@
+        
+        crt_to = [JuMP.@NLexpression(pm.model, ( pt_to[idx]*vr_to_pn[idx]+qt_to[idx]*vi_to_pn[idx])/(vr_to_pn[idx]^2+vi_to_pn[idx]^2)) for idx in 1:P]
+        cit_to = [JuMP.@NLexpression(pm.model, (-pt_to[idx]*vi_to_pn[idx]+qt_to[idx]*vr_to_pn[idx])/(vr_to_pn[idx]^2+vi_to_pn[idx]^2)) for idx in 1:P]
+
+        pt_bus_to_unmerged = [
+            [JuMP.@NLexpression(pm.model, vr_to[p]*crt_to[idx]+vi_to[p]*cit_to[idx]) for (idx,p) in enumerate(phases)]...,
+            JuMP.@NLexpression(pm.model, vr_to[n]*sum(-crt_to[idx] for idx in 1:P)+vi_to[n]*sum(-cit_to[idx] for idx in 1:P))
+        ]
+        qt_bus_to_unmerged = [
+            [JuMP.@NLexpression(pm.model, -vr_to[p]*cit_to[idx]+vi_to[p]*crt_to[idx]) for (idx,p) in enumerate(phases)]...,
+            JuMP.@NLexpression(pm.model, -vr_to[n]*sum(-cit_to[idx] for idx in 1:P)+vi_to[n]*sum(-crt_to[idx] for idx in 1:P))
+        ]
+    end
+    var(pm, nw, :pt_bus)[t_idx] = pt_bus_to = _merge_bus_flows(pm, pt_bus_to_unmerged, t_connections)
+    var(pm, nw, :qt_bus)[t_idx] = qt_bus_to = _merge_bus_flows(pm, qt_bus_to_unmerged, t_connections)
+end
+
+
+"""
+	function constraint_mc_transformer_power_rating(
+		pm::AbstractExplicitNeutralACRModel,
+		nw::Int,
+		id::Int,
+		f_idx::Tuple,
+		t_idx::Tuple,
+		f_bus::Int,
+		t_bus::Int,
+		f_connections::Vector,
+		t_connections::Vector,
+		config::ConnConfig,
+		sm_ub::Real;
+		report::Bool=true
+	)
+
+For ACR models with explicit neutrals,
+imposes a bound on the magnitude of the total apparent power at each winding of the transformer.
+
+```
+sum(pt_fr)^2 + sum(qt_fr)^2 <= sm_ub^2
+sum(pt_to)^2 + sum(qt_to)^2 <= sm_ub^2
+```
+"""
+function constraint_mc_transformer_power_rating(pm::AbstractExplicitNeutralACRModel, nw::Int, id::Int, f_idx::Tuple, t_idx::Tuple, f_bus::Int, t_bus::Int, f_connections::Vector, t_connections::Vector, config::ConnConfig, sm_ub::Real; report::Bool=true)
+    pt_fr = var(pm, nw, :pt, f_idx)
+    qt_fr = var(pm, nw, :qt, f_idx)
+    pt_to = var(pm, nw, :pt, t_idx)
+    qt_to = var(pm, nw, :qt, t_idx)
+
+    if sm_ub < Inf
+        JuMP.@constraint(pm.model, sum(pt_fr)^2 + sum(qt_fr)^2 <= sm_ub^2)
+        JuMP.@constraint(pm.model, sum(pt_to)^2 + sum(qt_to)^2 <= sm_ub^2)
+    end
+end
+
+
+# BRANCH
+
+# BRANCH - Variables
+
+"""
+	function variable_mc_branch_power(
+		pm::AbstractExplicitNeutralACRModel;
+		nw::Int=nw_id_default,
+		bounded::Bool=true,
+		report::Bool=true,
+		kwargs...
+	)
+
+For ACR models with explicit neutrals,
+creates branch power variables `:p` and `:q` and placeholder dictionaries for the terminal power flows `:p_bus` and `:q_bus`.
+"""
+function variable_mc_branch_power(pm::AbstractExplicitNeutralACRModel; nw::Int=nw_id_default, bounded::Bool=true, report::Bool=true, kwargs...)
+    variable_mc_branch_power_active(pm, nw=nw, bounded=bounded, report=report; kwargs...)
+    variable_mc_branch_power_reactive(pm, nw=nw, bounded=bounded, report=report; kwargs...)
```

Comment:
Should use `real` and `imaginary` instead of `active` and `reactive` for consistency with other naming conventions